hack/podmansnoop

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+# @lint-avoid-python-3-compatibility-imports
+from __future__ import print_function
```

Comment:
I think we can drop that, python 2 is EOL

Reply:
Done :heavy_check_mark: